file.js

```diff
@@ -1,83 +1,107 @@
+/* eslint-disable no-plusplus */
```

Comment:
- [ ] You have done great so far, :+1: :+1: Your project looks great. You just need to make some changes to your project.
- - As required by Microverse, you are not meant to use any linter disable rule in your project. The linters are there to make sure your project follows the best practices. I'll suggest you remove this and fix the error. For this error you can update your loop to this `(let i = 0; i < hideCard.length; i += 1)`. :+1: :smile: